public/apps/configuration/panels/role-view/index-permission-panel.tsx

```diff
@@ -139,8 +139,8 @@ export function IndexPermissionPanel(props: IndexPermissionPanelProps) {
   return (
     <PanelWithHeader
       headerText={headerText}
-      headerSubText="Index permissions allow you to specify how users in this role can access the indices. You can restrict a role to a subset of documents in an index, 
-      and which document fields a user can see as well. If you use field-level security in conjunction with document-level security, 
+      headerSubText="Index permissions allow you to specify how users in this role can access the indices. You can restrict a role to a subset of documents in an index,
+      and which document fields a user can see as well. If you use field-level security in conjunction with document-level security,
```

Comment:
Will this change affect the text?

Reply:
It is a ts warning that there is a trailing empty space. Just fixed it along the way.